netgrasp/netgrasp.py

```diff
@@ -437,7 +437,8 @@ def log_event(mid, iid, did, rid, event, have_lock = False):
         ng.debugger.debug("entering log_event(%s, %s, %s, %s, %s, %s)", (mid, iid, did, rid, event, have_lock))
 
         # Only log events for which there are subscribers.
-        if ((ng.email["enabled"] and event in ng.email["alerts"]) or (ng.notification["enabled"] and event in ng.notification["alerts"])):
+        if (ng.email["enabled"] and event in ng.email["alerts"]) \
+            or (ng.notification["enabled"] and event in ng.notification["alerts"]):
```

Comment:
same per above; I've generally left long lines for the editor to handle; I'm not opposed, but it feels like a way to introduce accidental bugs